scripts/ci_build_and_test_in_container.sh

```diff
@@ -74,6 +76,7 @@ RUN_UNIT_TESTS=true
 RUN_INTEGRATED_TESTS=false
 TEST_CODE_STYLE=false
 TEST_DOCUMENTATION=false
+CODE_COVERAGE=false
```

Comment:
```suggestion
CODE_COVERAGE=0
```
If you directly set `CODE_COVERAGE` to `0` or `1` (line 107) then you won't have to use the fancy `-DENABLE_COVERAGE=$([[ "${CODE_COVERAGE}" = true ]] && echo 1 || echo 0)` and you'll be able to use `CODE_COVERAGE` directly.